sdk/face/azure-ai-vision-face/src/samples/java/com/azure/ai/vision/face/samples/FindSimilarWithLargeFaceList.java

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.vision.face.samples;
+
+import com.azure.ai.vision.face.administration.FaceAdministrationClientBuilder;
+import com.azure.ai.vision.face.administration.LargeFaceListClient;
+import com.azure.ai.vision.face.FaceClient;
+import com.azure.ai.vision.face.FaceClientBuilder;
+import com.azure.ai.vision.face.models.FaceTrainingResult;
+import com.azure.ai.vision.face.models.FaceDetectionModel;
+import com.azure.ai.vision.face.models.FaceDetectionResult;
+import com.azure.ai.vision.face.models.FaceFindSimilarResult;
+import com.azure.ai.vision.face.models.FaceRecognitionModel;
+import com.azure.ai.vision.face.models.FindSimilarMatchMode;
+import com.azure.ai.vision.face.samples.utils.ConfigurationHelper;
+import com.azure.ai.vision.face.samples.utils.Resources;
+import com.azure.ai.vision.face.samples.utils.Utils;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.BinaryData;
+import com.azure.core.util.polling.SyncPoller;
+
+import java.util.List;
+
+import static com.azure.ai.vision.face.samples.utils.Utils.log;
+
+public class FindSimilarWithLargeFaceList {
+    public static void main(String[] args) {
+        //Create LargeFaceList
+        String largeFaceListId = "lfl01";
+
+        //Create LargeFaceListClient
+        LargeFaceListClient largeFaceListClient = new FaceAdministrationClientBuilder()
+            .endpoint(ConfigurationHelper.getEndpoint())
+            .credential(new AzureKeyCredential(ConfigurationHelper.getAccountKey()))
```

Comment:
We'd prefer TokenCredential (azure-identity) in samples (as SFI want to avoid key credential if possible)